runtime/src/main/java/org/corfudb/runtime/view/ObjectBuilder.java

```diff
@@ -95,6 +96,8 @@ public T open() {
         log.info("ObjectBuilder: open Corfu stream {} id {}", streamName, streamID);
 
         try {
+            verify();
```

Comment:
All this custom code can be replaced by lombok builder. If you want to clean up this code I believe it had better use a Lombok builder. 
And also would be great if you get rid of `@data` annotation and setters in the ObjectBuilder class

Reply:
Ok.

Reply:
It took more than annotating the class, I removed some other unnecessary code. Please take a look again. 

There's some really strange usage that I'll clean up later. For example, some code expects to access the Builder's field members before the object is built, that's why the builder has getters.

Reply:
Getters are ok, setters not desired